e2e/parallel/networksAndTestnetModeFlow.test.ts

```diff
@@ -0,0 +1,207 @@
+  delayTime,
+  doNotFindElementByTestId,
+  executePerformShortcut,
+  findElementByTestId,
+  findElementByTestIdAndClick,
+  findElementByText,
+  getAllWindowHandles,
+  getExtensionIdByName,
+  getRootUrl,
+  getWindowHandle,
+  goBackTwice,
+  goToPopup,
+  goToTestApp,
+  importWalletFlow,
+  initDriverWithOptions,
+  navigateToSettingsNetworks,
+  querySelector,
+  waitAndClick,
+} from '../helpers';
+import { TEST_VARIABLES } from '../walletVariables';
+
+let rootURL = getRootUrl();
+let driver: WebDriver;
+
+const browser = process.env.BROWSER || 'chrome';
+const os = process.env.OS || 'mac';
+
+describe('Networks & Testnet Mode flows', () => {
+  beforeAll(async () => {
+    driver = await initDriverWithOptions({
+      browser,
+      os,
+    });
+    const extensionId = await getExtensionIdByName(driver, 'Rainbow');
+    if (!extensionId) throw new Error('Extension not found');
+    rootURL += extensionId;
+  });
+  afterAll(async () => await driver.quit());
+
+  it('should be able import a wallet via seed', async () => {
+    await importWalletFlow(driver, rootURL, TEST_VARIABLES.EMPTY_WALLET.SECRET);
+  });
+
+  it('should be able to toggle testnet mode', async () => {
+    await navigateToSettingsNetworks(driver, rootURL);
+    await delayTime('short');
+
+    await findElementByTestIdAndClick({ driver, id: 'testnet-mode-toggle' });
+    await delayTime('medium');
+    const testnetBar = await findElementByTestId({
+      driver,
+      id: 'testnet-bar',
+    });
+    expect(testnetBar).toBeTruthy();
+
+    await findElementByTestIdAndClick({ driver, id: 'testnet-mode-toggle' });
+    await delayTime('medium');
+    const testnetBar2 = await doNotFindElementByTestId({
+      driver,
+      id: 'testnet-bar',
+    });
+    expect(testnetBar2).toBeFalsy();
+  });
+
+  it('should be able to toggle testnet mode shortcut', async () => {
+    await findElementByTestIdAndClick({
+      driver,
+      id: 'testnet-mode-shortcut-toggle',
+    });
+  });
+
+  it('should go back to home', async () => {
+    await goBackTwice(driver);
+  });
+
+  it('should be able to connect to bx test dapp', async () => {
+    await delayTime('long');
+    await goToTestApp(driver);
+    const dappHandler = await getWindowHandle({ driver });
+
+    const button = await findElementByText(driver, 'Connect Wallet');
+    expect(button).toBeTruthy();
+    await waitAndClick(button, driver);
+
+    const modalTitle = await findElementByText(driver, 'Connect a Wallet');
+    expect(modalTitle).toBeTruthy();
+
+    const mmButton = await querySelector(
+      driver,
+      '[data-testid="rk-wallet-option-rainbow"]',
+    );
+    await waitAndClick(mmButton, driver);
+
+    const { popupHandler } = await getAllWindowHandles({
+      driver,
+      dappHandler,
+    });
+
+    await driver.switchTo().window(popupHandler);
+
+    await clickAcceptRequestButton(driver);
+
+    await driver.switchTo().window(dappHandler);
+    const topButton = await querySelector(
```

Comment:
could use `findElementByTestId` here

Reply:
yeah, i replaced all three instances of this locally and got `✓ should be able to connect to bx test dapp 16439ms`

Reply:
ok i copy pasted this from other tests and it doesn't seem to be working for me

<img width="551" alt="image" src="https://github.com/rainbow-me/browser-extension/assets/12115171/f8d23ca0-b3dc-4a21-a1b4-07eef3a1c358">

will skip for this PR